sungc/deglint.py

```diff
@@ -192,8 +298,34 @@ def get_band_list(self, meas_dict):
             raise Exception(
                 "Could not find any geotifs in '{0}'".format(self.group_path)
             )
+
         return bandList, bandNums, bandNames
 
+    def get_fmask_file(self):
+        """
+        Get the fmask file from self.meas_dict
+
+        Returns
+        -------
+        fmask_file : str
+            fmask filename
+
+        Raises
+        ------
+        Exception if fmask file is not found
+        """
+        fmask_file = None
+        for key in self.meas_dict.keys():
+            if key.lower().find("fmask") != -1:
+                fmask_file = self.group_path.joinpath(
+                    self.meas_dict[key]["path"]
+                )
+
+        if not fmask_file:
+            raise Exception("\nCould not find the fmask file")
```

Comment:
I'd condense this down as well.

e.g.

```
if "fmask" in self.meas_dict:
    fmask_file = self.group_path.joinpath(self.meas_dict["fmask"]["path"])
else:
    raise Exception("\nCould not find the fmask file")
```

Reply:
Thank you, I've made the requested change